src/main.cpp

```diff
@@ -2986,6 +2986,8 @@ bool CVerifyDB::VerifyDB(CCoinsView *coinsview, int nCheckLevel, int nCheckDepth
             } else
                 nGoodTransactions += block.vtx.size();
         }
+        if (ShutdownRequested())
+            return true;
```

Comment:
If you return false you get a dialog suggesting to do a full reindex.

The true just lies about it being completed successfully, but thats Ok since we know the app will exit very soon.